src/seedsigner/views/psbt_views.py

```diff
@@ -32,13 +33,20 @@ def run(self):
         seeds = self.controller.storage.seeds
         button_data = []
         for seed in seeds:
-            button_str = seed.get_fingerprint(self.settings.get_value(SettingsConstants.SETTING__NETWORK))
+            fingerprint = seed.get_fingerprint(self.settings.get_value(SettingsConstants.SETTING__NETWORK))
+            if isinstance(seed, BIP85ChildSeed):
+                # Indent the Child Seed Label for a better hierarchical view
+                icon = " " * (GUIConstants.EDGE_PADDING // 4) + FontAwesomeIconConstants.ARROW_RIGHT
+                label = f"#{seed.child_index}: " + fingerprint
```

Comment:
This probably needs to be wrapped for translation via `_()`. It's possible that other languages might not use the "#" sign (ya, but we are already using the HASHTAG icon... 🤷‍♂️) and maybe the colon could be an issue, too?